js/decentralizer.js

```diff
@@ -34,6 +34,17 @@ function initialLoad() {
                 $.getJSON(path.join(__dirname, "../databases/farms.json"), function (farms) {
                     $.getJSON(path.join(__dirname, "../databases/farms_definition.json"), function (farmsDefinition) {
 
+                        // Checking if the host is online, and append storageprice
+                        for (var i = 0; i < contracts.length; i++) {

```

Comment:
can you change you `for (var` to `for (let`? Thanks
(the reason is `var` is today not recommended (it is considered as global variable), and so `let` is preferred to local variable and `const` to global constant variable.